components/AppHeader.vue

```diff
@@ -104,7 +110,7 @@
                     'overflow-hidden',
                     'overflow-ellipsis',
                   ]"
-                >{{ currentAddress }}</div>
+                >{{ sessionWallet }}</div>
```

Comment:
But it seems that I can still read `My works` and contents in each work before I sign the message
e.g.
<img width="1334" alt="image" src="https://github.com/user-attachments/assets/bebd3e80-210e-4266-aa26-78de97313c24">
<img width="1186" alt="image" src="https://github.com/user-attachments/assets/29e78bdf-2956-43c5-a53e-f165aa983110">

Can we only consider a user is connected after they properly sign a message and has a valid token?

Reply:
[👔 Use session wallet to determine connected user](https://github.com/likecoin/app-like-co/pull/474/commits/57d3ef7ba6f1da73bb5519203359a8b310113637)